tests/e2e/fixtures.js

```
@@ -113,49 +113,49 @@ class AIFeedbackUtils {
-		await use(new Editor({ page }));
+	editor: async ( { page }, use ) => {
+		await use( new Editor( { page } ) );
 	},
-	pageUtils: async ({ page }, use) => {
-		await use(new PageUtils({ page }));
+	pageUtils: async ( { page }, use ) => {
+		await use( new PageUtils( { page } ) );
 	},
-	requestUtils: async ({}, use) => {
-		const requestUtils = await RequestUtils.setup({
+	requestUtils: async ( {}, use ) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)